### PR TITLE
Added documentation for the new sync config option

### PR DESCRIPTION
**Pull request by Robinfr**
Documentation for the new sync config option: "Nothing (preserve data)" that should be used in conjunction with the "Sync to device" action.

### CODE REVIEW

File: content/refguide/offline-first.md [hunk 1]
```diff
@@ -62,11 +62,14 @@ After the download process (and thus synchronization) is completed, the widgets
...
 
-Depending on the use-case, more fine-grained synchronization controls might be required. Therefore, an app's  synchronization configuration can be customized to override the default configuration. It is possible to limit what is downloaded by using [XPath Constraints](xpath-constraints), which is applied in addition to the security constraints.
+Depending on the use-case, more fine-grained synchronization controls might be required. Therefore, it is possible to change the download behaviour for an entity. You can choose between the following options:
 
-Furthermore, it is possible to disable downloads for an entity. This can be very useful in cases where the objects should only be uploaded, for example a `Feedback` entity.
+- All Objects, download all objects applying the regular security constraints.
+- By XPath, only download the objects that are matching the [XPath Constraints](xpath-constraints) in addition to the regular security constraints.
+- Nothing (clear data), don't download any objects automatically, but do clear the database when performing a synchronization. This can be useful in cases where the objects should only be uploaded, for example a `Feedback` entity.
```
Comment (ekrem-aktas) [inline content/refguide/offline-first.md:71]: "... do clear the database ..."

It feels like this option influences the whole offline db, what about something like: `... do clear the data stored in the database for this entity when...."

Comment (Robinfr) [reply content/refguide/offline-first.md:71]: Good point, will change it.

Comment (Robinfr) [reply content/refguide/offline-first.md:71]: Changed ✔️

**Comment by ConnorLand** [comment]
Hi @Robinfr and @ekrem-aktas . As I slacked Robin, I believe these changes to the different sync config options should go to this existing Navigation reference guide https://docs.mendix.com/refguide/navigation#customize (Section 4.3.2). Do you think that's best? 

If so, can you please copy/paste the text I proofed, drop is in the Navigation refguide, and then commit to this PR again? That way the new language will end up in the right place and I can merge it easily on release day.

**Comment by Robinfr** [comment]
Hmm it seems we have the same information in multiple places. I'm fine with moving it and referencing it from the Offline first guide? What do you think @ekrem-aktas ?

**Comment by ConnorLand** [comment]
> Hmm it seems we have the same information in multiple places. I'm fine with moving it and referencing it from the Offline first guide? What do you think @ekrem-aktas ?

Indeed we do want the info only in one place. However, it would have to stay in the Navigation refguide and be referenced by Offline First because documentation is tied to UI. These options are offered as a part of the navigation profile UI, so the documentation for those options needs to be in that location.

**Comment by ekrem-aktas** [comment]
> Indeed we do want the info only in one place. However, it would have to stay in the Navigation refguide and be referenced by Offline First because documentation is tied to UI. These options are offered as a part of the navigation profile UI, so the documentation for those options needs to be in that location.

This form is useful only in the context of offline synchronization, otherwise we have to explain what does an "Upload" or "Download" mean in the navigation documentation. It is a bit technical, and we explain them in the offline-first docs. We'll we changing this form in the near future, where there will be more options, more tied to context of synchronization.

I'd have a screenshot & short description of the form and refer to offline-first docs, where the options are detailed.

**Comment by ConnorLand** [comment]
> > Indeed we do want the info only in one place. However, it would have to stay in the Navigation refguide and be referenced by Offline First because documentation is tied to UI. These options are offered as a part of the navigation profile UI, so the documentation for those options needs to be in that location.
> 
> This form is useful only in the context of offline synchronization, otherwise we have to explain what does an "Upload" or "Download" mean in the navigation documentation. It is a bit technical, and we explain them in the offline-first docs. We'll we changing this form in the near future, where there will be more options, more tied to context of synchronization.
> 
> I'd have a screenshot & short description of the form and refer to offline-first docs, where the options are detailed.

**Comment by Robinfr** [comment]
@ConnorLand I've made the changes as requested. I've removed the options from the navigation and now refer to the offline first documentation for more information.

**Pull request merged**